tpgtools/templates/resource.go.tmpl

```diff
@@ -413,7 +428,12 @@ func resource{{$.PathType}}Delete(d *schema.ResourceData, meta interface{}) erro
 {{- end}}
 
 	log.Printf("[DEBUG] Deleting {{$.Type}} %q", d.Id())
-	if err := config.client{{$.ProductType}}DCL.Delete{{$.Type}}(context.Background(), obj); err != nil {
+	userAgent, err := generateUserAgentString(d, config.userAgent)
+	if err != nil {
+		return err
+	}
+	client := Create{{$.ProductType}}Client(config, userAgent, "")
```

Comment:
What do we need to do to get the billing project here? Passing `""` seems strange, maybe remove this parameter until we are ready to use it?

Reply:
It's another task I have in tandem to add support for billing project. It was a little throttled by figuring out if we should upstream it to the dcl or not. I can remove it